### PR TITLE
Allow collections to be set by the create endpoint

### PR DESCRIPTION
**Pull request by jcoyne**
And returned by the show endpoint.

## Why was this change made?

Fixes https://github.com/sul-dlss/google-books/issues/391

## Was the API documentation (openapi.yml) updated?
yes

## Does this change affect how this application integrates with other services?
yes

If so, please confirm change was tested on stage and/or test added to sul-dlss/infrastructure-integration-test.
it will be tested on stage.

### CODE REVIEW

File: openapi.yml [hunk 1]
```diff
@@ -1366,6 +1366,8 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Sequence'
+        isMemberOf:
```
Comment (justinlittman) [inline openapi.yml]: A description in the openapi would be appreciated, because I can't say it is intuitive to have collection membership in structural.

Comment (jcoyne) [reply openapi.yml]: I don't know that there's a way to do that given that `$ref` can't have a sibling.
https://swagger.io/docs/specification/using-ref/#sibling

Comment (justinlittman) [reply openapi.yml]: It can have a sibling, it will just be ignored. However, having a description will still help someone reading the openapi.

Comment (jcoyne) [reply openapi.yml]: Okay

**Pull request merged**